src/charts/line.js

```diff
@@ -793,6 +798,56 @@ define(function(require){
                     .attr('y2', height - margin.bottom - margin.top);
         }
 
+        /**
+         * Draws custom user-defined lines onto the chart
+         * @return void
+         */
+        function drawCustomLines(){
+            svg.select('.custom-lines-group')
+                .selectAll('line')
+                .remove();
+
+            let yValues = customLines.map(it => it.y);
+
+            let getColor = yValue => {
+                const definedColor = customLines.find(it => it.y === yValue).color;
+                if(definedColor) {
+                    return definedColor;
+                }
+
+                return 'grey';
+            }
+
+            //draw a horizontal line to extend x-axis till the edges
+            verticalLines = svg.select('.custom-lines-group')
+                .selectAll('line.custom-line')
+                .data(yValues)
+                .enter()
+                .append('line')
+                .attr('class', 'custom-line')
+                .attr('x1', (-xAxisPadding.left - 30))
+                .attr('x2', chartWidth)
+                .attr('y1', (d) => yScale(d))
+                .attr('y2', (d) => yScale(d))
+                .attr('stroke', (d) => getColor(d));
+
+            // draw the annotations right above the line at the right end of the chart
+            for(let line of customLines) {
+                if(!line.name) {
```

Comment:
It's just a matter of preference but I believe you can do this instead:
```
if (line.name) {
   svg.select(...)
}
```
In this case no `continue` is necessary.

Reply:
Ok, I usually stick for the other approach, but I like the idea to remove `continue` as well.